### ISSUE TITLE
Panning is doing inverse when sort is set to descending

### ISSUE THREAD

**Issue by mattijn** [vega/vega-lite#5413]
Given the following minimal spec:
```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "data": {"url": "data/cars.json"},
  "mark": "tick",
  "encoding": {
    "x": {"field": "Horsepower", "type": "quantitative", "sort": "descending"}
  },
  "selection": {
    "zoompan": {"type": "interval", "bind": "scales", "encodings": ["x"]}
  }
}
```
And this animated gif:
![reverse_panning gif](https://user-images.githubusercontent.com/5186265/65759511-69636900-e11b-11e9-8481-367117b5fdf0.gif)

I was hoping that the panning remains in line with the mouse movement. But it is not. Now, because of the `"sort": "descending"`, the panning-movement is the inverse of the panning.

**Comment by kanitw** [comment]
Thanks.   Also a related bug, using scale.reverse is similarly problematic:

```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
  "data": {"url": "data/cars.json"},
  "mark": "tick",
  "encoding": {
    "x": {"field": "Horsepower", "type": "quantitative", "scale": {"reverse": true}}
  },
  "selection": {
    "zoompan": {"type": "interval", "bind": "scales", "encodings": ["x"]}
  }
}
```

**Comment by zakjan** [comment]
I have this issue as well

**Comment by andreasplesch** [comment]
me too: https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAkDODGALApgWwIaQFxUQFzwAdYsB6UgN2QHN0A6agSz0QFcAjOxge1IRQyUa6ALQAbZskoAWOgCtY3AHaQANOCgATdHkw5gkVgCcx2LTvSl46I7HmKVAX3URIsZGOTw8PFfsjURoyaZgZ4AJ6EyGaQjEp4yEYU6KaqUOxxIThu1p6wkI7OGpAYRgDWMfCMRvCeasXISvDcmnHUoRquAB4dEH1QAGaMHllQABLctsiE3ADuifX9rhFRMQCOrOjxzDqMVItLOSnR-prcGHFmANoA7ACsaQCMdwAMALpFS1CKRngxmsgEI1Wkp2p0wJ8+pBwr1DkMRjEALKMPIAfSiRlRAHEUmJlAd+pAVicoBstj5dD59i5DghjqFIGcLn4wFcAEwvNLSd6Qwk-P7ZAFApQgsH9XnfRgALxJBnhYlGkAAwuEJCLEvk0kTIiTIGTtpS9tFHBoTY4gA

**Comment by arvind** [comment]
An additional spec from #5386:

```json
{
  "data": {"url": "data/cars.json"},
  "mark": "point",
  "encoding": {
    "x": {
      "type": "quantitative",
      "field": "Horsepower",
      "scale": {"domain": [250, 0]}
    },
    "y": {"type": "quantitative", "field": "Miles_per_Gallon"}
  },
  "selection": {
    "pan": {"type": "interval", "bind": "scales"}
  }
}
```

**Comment by arvind** [comment]
Fixed with #7163.
